packages/sui-js/README.md

```diff
@@ -0,0 +1,17 @@
+# sui-js
+> Set of useful js utilities
+
+```sh
+$ npm install @s-ui/js --save-dev
+```
+
+## browser
+A browser detector (device, browser, OS...).
+Imports [https://github.com/lancedikson/bowser](https://github.com/lancedikson/bowser).
+
+
+```js
+import browser from '@s-ui/js/lib/browser'
```

Comment:
Does it make sense to name it browser if its a bowser thing ?
I mean if we add somday axios I assume we'll name it axios, not fetcher, isn't it ?

Reply:
from my point of view, today we are using bowser but tomorrrow we can use any other package. since this is browser related, i would keep using this name.

Reply:
It makes sense as it's decoupled of the library we're using. We might use tomorrow another library or even create our own behind this name. That would be the idea of this package, being a black box that, with exports an API to get the browser info, but the developer doesn't need to know what's happening under the hood.

If someday we use axios, we will do a ' @s-ui/js/lib/fetcher'.

Reply:
I agree with the principle. But usually when you do that you create a façade with only the methods you wanna expose, not full access to a thirdparty API. This way if you change the library, you know exactly what you need to maintain. 
That's why I was asking. Either you expose expressively a thirdparty lib, either you limit the coupling to what you expose.

Reply:
Makes sense. Let me refactor some code and we'll take a look again.

Reply:
Done. Take a look @davidbarna